lib/ntp.js

```diff
@@ -136,7 +136,7 @@ function getLogger() {
   } else {
     return function(message) {
       var canLogKadira =
-        Meteor._localStorage.getItem('LOG_KADIRA') !== null
+        global?.localStorage?.getItem('LOG_KADIRA') !== null
```

Comment:
I'm wondering if we can remove this logger. We haven't had any need for it in over 3 years.

If we keep it, accessing localStorage should be wrapped within try ... catch since older browsers will sometimes throw in `getItem`. Also, if local storage is not available, it should default to not logging to match the previous behavior.